packages/babel-generator/test/index.js

```diff
@@ -348,6 +348,20 @@ describe("generation", function () {
     const output = generate(type).code;
     expect(output).toBe("(infer T)[]");
   });
+
+  it("comments with same start index", () => {
```

Comment:
Can we describe what we're actually testing here? i.e. "should not deduplicate comments with same start index from two different ASTs" or something similar.